test/index.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-env mocha */
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const test = require('ava');
+const stringifyObject = require('../');
```

Comment:
You can use `import` syntax here as test files are transpiled with Babel automatically. You can also drop line 1 and 2.

Example: https://github.com/sindresorhus/indent-string/blob/458eca3f626b95bdcff5afe30d1568bf76889920/test.js#L1